zebra-state/src/constants.rs

```diff
@@ -18,7 +18,7 @@ pub use zebra_chain::transparent::MIN_TRANSPARENT_COINBASE_MATURITY;
 pub const MAX_BLOCK_REORG_HEIGHT: u32 = MIN_TRANSPARENT_COINBASE_MATURITY - 1;
 
 /// The database format version, incremented each time the database format changes.
-pub const DATABASE_FORMAT_VERSION: u32 = 14;
+pub const DATABASE_FORMAT_VERSION: u32 = 15;
```

Comment:
Database format changes are risky changes, so we want to wait until after the testnet rollback release to merge this PR.

Reply:
(At the engineering sync, we decided not to wait for the rollback, because it might take a few more days.)